internal/importers/godep/godep_importer.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
```

Comment:
is it a bit redundant to name the file `godep_importer.go` if we're already in a package/dir named `godep`?